watertap/unit_models/tests/test_gac.py

```diff
@@ -52,11 +53,16 @@
 )
 from watertap.costing import WaterTAPCosting
 
-
 __author__ = "Hunter Barber"
 
 solver = get_solver()
 
+# inputs for badly_scaled_var_generator used across test frames
+sv_large = 1e2
+sv_small = 1e-2
+sv_zero = 1e-8
+
+print(isinstance({"species": "TCE"}, list))
```

Comment:
This could be that Github is hiding some code that I'm missing, but why is this here? Seems out of place.

Reply:
Nope, this is an accident. Removing it.